easy_infra.yml

```diff
@@ -45,20 +52,22 @@ packages:
     version: 2.2.125
     version_argument: --version
   consul-template:
-    helper: ["all"]
+    helper:
+    - all
     version: v0.29.5
     version_argument: --version
   envconsul:
-    helper: ["all"]
+    helper:
+    - all
     version: v0.13.1
     version_argument: --version
   fluent-bit:
-    helper: ["all"]
+    helper:
+    - all
     version: v2.0.5
     version_argument: --version
   kics:
```

Comment:
All changes to this file other than this section are simply reformatting changes.  We are only updating the `kics` version, and removing the `allow_update: false`